docs/checks.md

```diff
@@ -141,7 +141,9 @@ Risk: `Low` (possible unknown vulnerabilities)
 
 This check tries to determine if the project runs tests before pull requests are
 merged. It is currently limited to repositories hosted on GitHub, and does not
-support other source hosting repositories (i.e., Forges).
+support other source hosting repositories (i.e., Forges). This check only considers
+and rewards repositories that do run test. Repositories, however, will not be penalized 
+if they do not run test before pull requests.  
```

Comment:
I'm not sure this reflects the clarification #3616 was trying to make. I believe the distinction was that tests can pass or fail, and Scorecard only considers tests that succeed/pass, not that there are tests present.

> Repositories, however, will not be penalized if they do not run test before pull requests.  

Repositories **are** penalized if they don't run tests, so this addition should be removed.


Also please see the CONTRIBUTING.md for instructions on how to modify this file
https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#updating-docs